doc/TeamsConnector.md

```diff
@@ -0,0 +1,81 @@
+# reshuffle-microsoft-connectors
+
+[Code](https://github.com/reshufflehq/reshuffle-microsoft-connectors) |
+[npm](https://www.npmjs.com/package/reshuffle-microsoft-connectors) |
+[Code sample](https://github.com/reshufflehq/reshuffle/tree/master/examples/microsoft/teams)
+
+`npm install reshuffle-microsoft-connectors`
+
+### Reshuffle Microsoft Connectors
+
+This package contains a collection of [Reshuffle](https://github.com/reshufflehq/reshuffle)
+connectors to [Microsoft](https://microsoft.com).
+
+The following example adds a new worksheet to an excel file
+
+```js
+const { Reshuffle } = require('reshuffle')
+const { TeamsConnector } = require('reshuffle-microsoft-connectors')
+
+const app = new Reshuffle()
+const connector = new TeamsConnector(app, { AppId, AppPassword, AppTenantId })
```

Comment:
I think we should keep the convention of `process.env` for arbitrary parameters, WDYT?